misc/juliacon2020.md

```diff
@@ -0,0 +1,90 @@
+- This part will be of interest of the majority of possible audiences and so should contribute to the whole Julia community
+
+2. Describe implementation
+- In this part, I would like to explain how we solve/implement problems and features; it can help other existing coding environments improve or encourage someone to create another Juno-like tool, or back to our project, attract future contributors and make it easier for them to contribute to Juno
+- Possible presentation flow:
+  * How to separate Juno packages from user's environment: maybe help julia-vscode extension, for example ?
+  * How to implement linter/refactor using both static code analysis and user runtime info: if we can implement "good" features using information that lives in user's runtime, it will show the benefits of Juno's approach in comparison to an ordinal IDE approach based on static code analysis 
+
+Each part will take some time (say, around 10 min), so I think our presentation form is better to be "Talk" (30 mins long including 5 mins for questions) rather than "Lightning Talk" (5 mins long).
+
+## Body
+
+We need to write title/abstract/description to fulfill a submission.
+
+### Submission title
+
+Juno 1.0 – the powerful IDE has got yet more power !
+
+(53/64 chars)
+
+### Submission type
+
+Talk
+
+### Abstract
+
+We will introduce Juno 1.0 -- show how it will get rid of the longstanding issues and bring us huge productivity boosts. 
+Juno's unique approach allows IDE features to be implemented simply but yet with the power of user runtime.
+
+(232/500 chars)
+
+### Description
+
+[Juno](https://junolab.org/) is an IDE for Julia.
+It enables unique, powerful and very interactive development style, and has been widely used in the community since its first publication in 2014.
+But due to the limitations that comes from its package infrastructure, there also have been some longstanding and fundamental problems mostly around installation, update, activation time, and interference with user's own package environment.
+
+In this presentation we are going to present Juno 1.0[^not-yet!], explain how it will solve those problems and improve user experiences, and also showcase our latest new features that are being implemented by using both static code analysis technique and user's runtime information.
+
+We also plan to describe our approach to solve the difficult problems and implement the new features; some parts of our approach can be applicable to other development tools and help them improve, as like we actually borrow the efforts devoted to [julia-vscode extension](https://www.julia-vscode.org/) for our static code analysis.
+
+[^not-yet!]: NOTE: I'm writing this 4 months ahead of time; you know, we've not done the work yet ! If you're interested in contribution, hit us on our slack channel. We would really appreciate your help.
+
+(1272/2500 chars)
+
+### Notes (optional)
```

Comment:
I'm thinking about leaving a note that says I want to take a part in JSoC or an intern job for this project, in this section.
What do you think on this idea ?